variables.tf

```diff
@@ -23,8 +23,10 @@ variable "hec_acknowledgment_timeout" {
 }
 
 variable "hec_token" {
-  description = "KMS encoded Splunk hec token"
+  description = "Pass the HEC token in plain text (not recommended) or through a parameter store, through a KMS encryption module, etc..."
   type        = string
+  default     = null
```

Comment:
Doesn't it make more sense to make it `nullable = false` ?

Reply:
indeed.

Fixed in f66b9b0 so that it is mandatory to pass it.

`nullable = false` needs terraform 1.1.0 and it would make sense to add it to more fields, I'll take a look in a subsequent MR

Reply:
@rporres +1 
I want to resolve the thread, but I can't. 
Lets give it a try permission-wise? :)

Reply:
I'll take a look into the permissions.